src/models/user.ts

```diff
@@ -97,8 +97,26 @@ const UserRole = mongoose.model(
     })
 )
 
+const organisationSchema = new Schema({
+    orgEmail:{
+        type: String
+    },
+    orgName:{
```

Comment:
this can be just `name` since we already have the name as organization.

You can eliminate the double reference.
Also, we should be having an organization owner email